### DISCUSSION TITLE
Issues/parse errors

### DISCUSSION THREAD

**Discussion by Trueset72** [vercel/next.js#19037] [Help]
During Next.js tutorial I accidently deleted a part of the "index.js" file. I then found the original code and wrote it all back out. For whatever reason , even with the container declared <div className="container"> , it is still showing up as parse error. When I type .container { its wrong on the ".container" part and when I delete the action "." It says the "{" is wrong. Can someone please help with this?

**Comment by Ihatetomatoes** [comment]
Which tutorial are you following? Do you have a repo with your code?

**Reply by Trueset72** [reply]
I'm following the "vercel/ next-learn-starter"  I believe I had to create a repo at the very beginning.

**Reply by Trueset72** [reply]
I'm at the point of linking pages with "Link" command

**Reply by Ihatetomatoes** [reply]
If you push your code to Github I can have a look.

Otherwise just follow the standard debugging. Remove as much of the content from `index.js` and keep adding it slowly in until it breaks, then you will know which line is the issue.

**Reply by Trueset72** [reply]
Question, how do I push? Lol

**Reply by Ihatetomatoes** [reply]
It's your first time using git?

This guide might help you:
https://docs.github.com/en/free-pro-team@latest/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line

Or if you are using VSCode, check out this video.

https://www.youtube.com/watch?v=F2DBSH2VoHQ

**Reply by Trueset72** [reply]
Yeah, kind of new to mostly all of it. I appreciate it though
